README.md

```diff
@@ -16,3 +16,5 @@ the function and run automatically when you commit through a GitHub action.
 ## Runtime Analysis
 
 What is the worst-case big $\Theta$ time complexity of your algorithm?
+
+Answer: The worst case $\Theta$ time complexity for my algorithm would be $\Theta((V^2)!)$. In this case, the isomorphic helper function would iterate through every vertex in graph1 and graph2 for every permutation of the possible mappings. 
```

Comment:
This seems unnecessarily inefficient. Don't you only need all permutations of one of the graphs? And please don't open multiple pull requests.

Reply:
I suppose it is unnecessarily inefficient. Sorry I had to open multiple pull requests when I was trying to get my tests to run automatically through github. I am not sure if I will be able to optimize this in time. It took me about a week to get the code to work the way it is.

Reply:
Doing permutations for only one graph should make the code much easier.